### DISCUSSION TITLE
Welcome to dcs-bios-arduino-library Discussions!

### DISCUSSION THREAD

**Discussion by charliefoxtwo** [talbotmcinnis/dcs-bios-arduino-library#76] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JackFlight27** [comment]
Is it possible to use multiplexers (74HC4067) with DCS bios fork? If that is possible, how should the code for that look like?
